styles/widgets/android5/switch.android5.less

```diff
@@ -39,13 +39,13 @@
 }
 
 .dx-switch-inner {
-    width: 100%;
+    width: 200%;
```

Comment:
Why 200 percent?

Reply:
New logic  means the same layout for all themes.